benchmarks/dynamo/benchmarks.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+from typing import Set
+
+# Note - hf and timm have their own version of this, torchbench does not
+# TOOD(voz): Someday, consolidate all the files into one runner instead of a shim like this...
+def model_names(filename: str) -> Set[str]:
+    names = set()
+    with open(filename, "r") as fh:
+        lines = fh.readlines()
+        lines = [line.rstrip() for line in lines]
+        for line in lines:
+            line_parts = line.split(" ")
+            if len(line_parts) == 1:
+                line_parts = line.split(",")
+            model_name = line_parts[0]
+            names.add(model_name)
+    return names
```

Comment:
why is this not just

```
{row[0] for row in csv.reader(fh)}
```

Reply:
Cause I didnt know about this, also, less readable?

Reply:
> less readable

🤔